bindings/python/multibody/model.hpp

```diff
@@ -156,8 +156,6 @@ namespace se3
         .def("createData",&ModelPythonVisitor::createData)
         .def("BuildEmptyModel",&ModelPythonVisitor::maker_empty)
```

Comment:
You can also remove this lines and put them as deprecated.
They are indeed redundant with respect to `pinocchio.Model()`

Reply:
You mean, I should also remove `model.createData()`? Or only `Model.BuildEmptyModel()`?

Reply:
`Model.BuildEmptyModel()` is the static method to remove.

Reply:
Perfect